src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $basicWhite;
+  padding: 0 50px;
+
+  &__logo-link {
+    display: flex;
+  }
+
+  &__logo-img {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__navigation {
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &__nav-item {
+    margin: 0 10px;
+  }
+
+  &__nav-link {
+    display: inline-block;
+    height: 60px;
+    line-height: 60px;
+    font-family: $fontRoboto;
+    color: $mainAccent;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  :first-child {
+    margin-left: 0;
+  }
+
+  :last-child {
+    margin-right: 0;
+  }
+
+  :hover {
+    color: $accentColour;
+  }
```

Comment:
The `:hover` selector is not scoped to any specific element within the `.header` class. It should be applied to a specific element, such as `.header__nav-link:hover`, to ensure it targets the correct elements.